packages/react-ui-ag/src/Modals/MoreOptionsModal.js

```diff
@@ -0,0 +1,266 @@
+    } else if (count === 1) {
+      propertyCount = '1 Property Found'
+    }
+
+    return propertyCount
+  }
+  @autobind
+  handleOutsideMouseDown(e) {
+    // If a click starts outside the modal, set the outside flag
+    // so when the click event occurs on the overlay,
+    // it will close the modal.
+    this.clickedOutside = (e.target === this.overlay)
+  }
+
+  @autobind
+  handleOutsideMouseUp(e) {
+    // If a click ends inside the modal, unset the outside flag
+    // so when the click event bubbles up to the overlay,
+    // it will not close the modal.
+    if (e.target !== this.overlay) {
+      this.clickedOutside = false
+    }
+  }
+
+  @autobind
+  handleOutsideClick(e) {
+    const { onClose, applyFilters } = this.props
+
+    // Verify the click started on the overlay.
+    // This will handle the case of the user starting a click inside the modal,
+    // then dragging and releasing the button outside the modal.
+    if (!this.clickedOutside) {
+      return
+    }
+
+    // Clear the flag so it will not affect the next click
+    this.clickedOutside = null
+
+    // Verify the click ended on the overlay and did not bubble up
+    // from the modal
+    if (e.target !== this.overlay) {
+      return
+    }
+
+    if (onClose) {
+      onClose()
+      if (applyFilters) {
+        applyFilters()
+      }
+    }
+  }
+
+  @autobind
+  handleSubmit(e) {
+    const {
+      onSubmit,
+      onClose,
+    } = this.props
+
+    if (!onSubmit) {
+      e.preventDefault()
+    } else {
+      onSubmit()
+    }
+
+    if (onClose) {
+      onClose()
+    }
+  }
+
+  @autobind
+  handleCancel() {
+    const {
+      onClose,
+      restoreFilters,
+    } = this.props
+    restoreFilters()
+    onClose()
+  }
+
+  formatNumberWithCommas(number) {
+    return String(number).replace(/(.)(?=(\d{3})+$)/g, '$1,')
+  }
+
+  renderHeader() {
+    const {
+      onClose,
+      theme,
+    } = this.props
+
+    const propertyCount = this.getPropertyCount()
+
+    return (
+      <div className={theme.MoreOptionsModal_Header}>
+        <h2
+          className={theme.MoreOptionsModal_PropertyCount}
+          data-tid="more-filters-props-count"
+        >
+          {propertyCount}
+        </h2>
+        <div
+          role="button"
+          tabIndex={0}
+          className={theme.MoreOptionsModal_CloseButton}
+          onClick={onClose}
+          data-tid="more-filters-close"
+          data-tag_item="close"
+        >
+          Close
+        </div>
+      </div>
+    )
+  }
+
+  renderFooter() {
+    const {
+      theme,
+      clearFilters,
+    } = this.props
+
+    const showPropertyButton = this.getShowPropertyButton()
+
+    return (
+      <div className={theme.MoreOptionsModal_Footer}>
+        <Button
+          className={theme.MoreOptionsModal_ResetAll}
+          onClick={clearFilters}
+        >
+          <span
+            data-tid="more-filters-reset-all"
+            data-tag_item="reset_all"
```

Comment:
Why can’t this be put on the button instead?

Reply:
@tadjohnston  I am actually going to remove the tagging for now because they have not set the tagging requirements for this card yet. I should probably wait to see how they want to tag everything. I will remove the extra span since they are not needed. And use the button component throughout